### PR TITLE
feat(testability): Expose function frameworkStabilizers

### PR DESCRIPTION
**Pull request by hankduan**

### CODE REVIEW

**Comment by hankduan** [comment]
Julie, can you take a look?

File: modules/angular2/src/core/testability/testability.ts [hunk 1]
```diff
@@ -16,6 +16,8 @@ export class Testability {
   /** @internal */
   _pendingCount: number = 0;
   /** @internal */
+  _didWork: boolean = false;
```
Comment (juliemr) [inline modules/angular2/src/core/testability/testability.ts]: A comment on what this is for might be useful for someone unfamiliar with the goal of our framework stabilizers.

**Comment by juliemr** [comment]
Sorry for the slow review! Looks like the dart tests are failing. Other than that looks ok.

**Comment by juliemr** [comment]
Also, modify tests at https://github.com/angular/angular/blob/master/modules/angular2/test/core/testability/testability_spec.ts to look for `didWork` please!

**Comment by hankduan** [comment]
Added comment for didWork and added tests. Still trying to figure out what's wrong with the dart tests ...

**Comment by hankduan** [comment]
Still can't figure out what is wrong and angular does not build on my local machine properly, so it's making life difficult. 
I'll come back to this in week of 12/14

**Comment by hankduan** [comment]
Ok...slight (although not very useful) update. Everything passes when I run the e2e tests locally =)

**Comment by juliemr** [comment]
This looks good to me! Assigning to @tbosch for a quick sanity check. cc @goderbauer

File: modules/angular2/src/platform/browser/testability.ts [hunk 2]
```diff
@@ -48,6 +48,25 @@ export class BrowserGetTestability implements GetTestability {
...
+          callback(didWork);
+        }
+      };
+      testabilities.forEach(function(testability) { testability.whenStable(decrement); });
+    };
+
+    if (!global.frameworkStabilizers) {
+      global.frameworkStabilizers = ListWrapper.createGrowableSize(0);
+    }
+    global.frameworkStabilizers.push(whenAllStable);
```
Comment (tbosch) [inline modules/angular2/src/platform/browser/testability.ts]: @juliemr Can we write a test that uses `global.frameworkStabilizers`?

**Comment by tbosch** [comment]
Looks good as well. Would be nice to have a test for the new feature though... (i.e. access `global.frameworkStabilizers` from an e2e test...

**Comment by hankduan** [comment]
Added e2e test

**Comment by tbosch** [comment]
Looks good!

**Comment by hankduan** [comment]
squashed both commits together

**Pull request closed without merging**